x-pack/plugins/infra/server/lib/alerting/common/utils.ts

```diff
@@ -134,3 +141,64 @@ export const getAlertDetailsUrl = (
   spaceId: string,
   alertUuid: string | null
 ) => addSpaceIdToPath(basePath.publicBaseUrl, spaceId, `/app/observability/alerts/${alertUuid}`);
+
+export const NUMBER_OF_DOCUMENTS = 10;
+export const KUBERNETES_POD_UID = 'kubernetes.pod.uid';
+export const termsAggField: Record<string, string> = { [KUBERNETES_POD_UID]: 'container.id' };
```

Comment:
use Tech Fields instead of hardcoded strings?

Reply:
Same as above comment. We don't have these fields defined in rule registry. And this change is related to one rule only at the moment, so I think it would be better to keep them within `infra` plugin.